File: src/main/java/io/stouder/adonis/edge/psi/impl/EdgeMustacheImpl.java
package io.stouder.adonis.edge.psi.impl;

import com.intellij.lang.ASTNode;
import io.stouder.adonis.edge.psi.EdgeMustache;
import org.jetbrains.annotations.NotNull;

public class EdgeMustacheImpl extends EdgePsiElementImpl implements EdgeMustache {
    public EdgeMustacheImpl(@NotNull ASTNode node) {
        super(node);
    }
}

File: src/main/java/io/stouder/adonis/edge/psi/impl/EdgeMustacheImpl.kt
package io.stouder.adonis.edge.psi.impl

import com.intellij.extapi.psi.ASTWrapperPsiElement
import com.intellij.lang.ASTNode
import io.stouder.adonis.edge.psi.EdgeMustache
import io.stouder.adonis.edge.psi.EdgePsiElement

class EdgeMustacheImpl(node: ASTNode) : ASTWrapperPsiElement(node), EdgeMustache, EdgePsiElement
